src/sagas/pushNotification.js

```diff
@@ -559,12 +585,14 @@ export function* resetPushNotification() {
   yield STORE.removeItem(pushNotificationKey.deviceId);
   // Reset the state
   yield put(pushReset());
-  yield fork(onAppInitialization);
+  yield put(initPushNotification());
+  console.log('Push notification reset successfully');
 }
 
 export function* saga() {
   yield all([
-    fork(onAppInitialization),
+    debounce(500, [[types.START_WALLET_SUCCESS, types.INIT_PUSH_NOTIFICATION]], init),
```

Comment:
Because of the feature flag update. The push notification can start the init first, and very soon after it, the feature flag can update and call the init again.